staging/src/k8s.io/pod-security-admission/policy/check_restrictedVolumes.go

```diff
@@ -122,6 +122,8 @@ func restrictedVolumes_1_0(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSp
 				badVolumeTypes.Insert("rbd")
 			case volume.FlexVolume != nil:
 				badVolumeTypes.Insert("flexVolume")
+			case volume.Cinder != nil:
+				badVolumeTypes.Insert("cinder")
```

Comment:
fortunately, because this was coded defensively, it would not have actually allowed the cinder volumes, it just would have rejected them with a less helpful "unknown volume type disallowed" error